src/styles.css

```
@@ -274,10 +342,38 @@
+      transform: translateX(0);
+    }
+  }
+
   @media (prefers-reduced-motion: reduce) {
-    ::view-transition-old(.main-content-transition),
-    ::view-transition-new(.main-content-transition) {
+    ::view-transition-old(main-content-transition),
+    ::view-transition-new(main-content-transition),
+    html[data-route-transition-fallback] main {
       animation-duration: 0.01ms !important;
+      animation: none !important;
```

Stylelint (17.9.0)

[error] Overridden property "animation-duration" by shorthand "animation" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)